packages/ez-dev/storybook/data.ts

```diff
@@ -9,30 +9,35 @@ export const dimensions = {
 export const rawData: RawData = [
   {
     value: 9,
+    value1: 20, 
     name: 'Alpha',
     id: '1',
     v: 2,
   },
   {
     value: 45,
+    value1: 20, 
```

Comment:
@IlyesBenAmara Could we add a value2 as well and update value1 to make some variations in height ?